### PR TITLE
fix(vue-component): [mind-map] border-radius & border

### PR DESCRIPTION
**Pull request by GaoNeng-wWw**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/tiny-vue/blob/dev/CONTRIBUTING.md)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: #1500 

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Comment by GaoNeng-wWw** [comment]
Snipaste_2024-03-26_10-29-16

demo ^^^

File: packages/vue/src/mind-map/src/pc.vue [hunk 1]
```diff
@@ -27,3 +27,35 @@ export default defineComponent({
   }
 })
 </script>
+
+<style lang="less">
```
Comment (zzcr) [inline packages/vue/src/mind-map/src/pc.vue]: 你好，我们的样式需要在vue-theme工程中编写，并且需要符合组件库类名的命名规范

Comment (GaoNeng-wWw) [reply packages/vue/src/mind-map/src/pc.vue]: 好的，我修改下

**Review by AcWrong02 — APPROVED** [review]

File: packages/theme/src/mind-map/index.less [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+        border: unset;
+        box-shadow: unset;
+
+        #input-box {
+            border-radius: var(--main-radius);
+            box-shadow: 0 0 0 4px var(--selected);
+            border: unset;
+        }
+    }
+}
```
Comment (kagol) [inline packages/theme/src/mind-map/index.less:33]: 代码格式化似乎有问题，应该是两个空格的缩进
@GaoNeng-wWw

Comment (GaoNeng-wWw) [reply packages/theme/src/mind-map/index.less:33]: 已修改

File: packages/theme/src/mind-map/index.less [hunk 3]
```diff
@@ -0,0 +1,33 @@
+.map-container {
```
Comment (zzcr) [inline packages/theme/src/mind-map/index.less:1]: 你好，意见TinyVue的class类名都按照现有的命名规范，以tiny-开头

Comment (GaoNeng-wWw) [reply packages/theme/src/mind-map/index.less:1]: image

这个类名，mind-elixir写死了

**Review by zzcr — APPROVED** [review]

**Pull request merged**